pkg/discovery/network/pods.go

```diff
@@ -49,10 +50,10 @@ func findPodCommandParameter(clientSet kubernetes.Interface, labelSelector, para
 	return "", nil
 }
 
-func findPod(clientSet kubernetes.Interface, labelSelector string) (*v1.Pod, error) {
+func FindPod(clientSet kubernetes.Interface, labelSelector string) (*v1.PodList, error) {
 	pods, err := clientSet.CoreV1().Pods("").List(v1meta.ListOptions{
 		LabelSelector: labelSelector,
-		Limit:         1,
+		//Limit:         1,
```

Comment:
@mangelajo Are these changes and the relevant changes throughout the code ok?

Reply:
I can change the func name to FindAllPods or FindPods

Reply:
I suggest `FindPods`.